Ballerina-Training/Ballerina-Forum-Application/back-end/bal-forum/types.bal

```diff
@@ -0,0 +1,193 @@
+import ballerina/http;
+import ballerina/time;
+
+# Represents the new user request
+type NewUser record {|
+    # Name of the user
```

Comment:
What is the intention of each of these comments? Aren't those self -explanatory?

Reply:
Yes, these are self-explanatory. Will remove them

Reply:
Fixed via [1eae7ab](https://github.com/ballerina-guides/training-content/pull/12/commits/1eae7abbc8d1d6bdd3a677b4fac348875ee53fb7)